src/components/badges/List.tsx

```diff
@@ -67,33 +57,16 @@ function BadgeListSuspended() {
       <ConfettiIfNeeded />
       <Scrollbar>
         <div className={space('space-y-2')}>
-          <BadgeSection
-            title="Mainnet NFT derivatives"
-            minted={ownedExternalERC721DerivativeContracts}
-            proofs={proofsAvailableToMint.filter(
-              (proof) =>
-                proof instanceof ERC721Proof &&
-                proof.network === Network.Mainnet
-            )}
-            onMinted={onMinted}
-          />
-          <BadgeSection
-            title="Goerli NFT derivatives"
-            minted={ownedERC721DerivativeContracts}
-            proofs={proofsAvailableToMint.filter(
-              (proof) =>
-                proof instanceof ERC721Proof && proof.network === Network.Goerli
-            )}
-            onMinted={onMinted}
-          />
-          <BadgeSection
-            title="Email derivatives"
-            minted={ownedEmailDerivativeContracts}
-            proofs={proofsAvailableToMint.filter(
-              (proof) => proof instanceof EmailProof
-            )}
-            onMinted={onMinted}
-          />
+          {Object.entries(ownedContractsByKey).map(([key, contracts]) => (
+            <>
```

Comment:
I think it's better not to wrap `each` `BadgeSection`, but to wrap all `BadgeSection`s into `<></>`

Reply:
So move this line to `L60`